nautobot/ui/README.md

```diff
@@ -19,3 +19,10 @@ Start Nautobot on port `8080/tcp`:
 ```no-highlight
 nautobot-server runserver 0.0.0.0:8080 --insecure
 ```
+
+## Running test
+
+To run all test suite, cd into ui directory and run 
+```
+npm run test
```

Comment:
Can we get an invoke command that can run this inside the containers too? It should honor invoke local and run it locally if you are setup that way.